tensorrt_llm/llmapi/llm_args.py

```
@@ -1988,6 +2005,11 @@ class TorchLlmArgs(BaseLlmArgs):
          Note that each CUDA graph can use up to 200 MB of extra memory.",
         status="beta")
 
+    attention_dp_config: Optional[AttentionDpConfig] = Field(
```

Ruff (0.12.2)

Line too long (138 > 120)

(E501)